CEmu.pro

```diff
@@ -12,22 +12,23 @@ TEMPLATE = app
 TRANSLATIONS += i18n/fr_FR.ts
 
 CONFIG += c++11
-
-GLOBAL_FLAGS = -W -Wall -Wno-unused-parameter -Werror=shadow -Werror=write-strings -Werror=redundant-decls -Werror=format -Werror=format-security -Werror=implicit-function-declaration -Werror=date-time -Werror=missing-prototypes -Werror=return-type -Werror=pointer-arith -fno-strict-overflow -Winit-self -ffunction-sections -fdata-sections
-
-if (macx | linux) {
-    GLOBAL_FLAGS += -fsanitize=address,bounds -fsanitize-undefined-trap-on-error -fstack-protector-all -Wstack-protector --param=ssp-buffer-size=1 -fPIC
-}
-if (macx) {
-    MORE_LFLAGS += -Wl,-dead_strip
+!win32 {
```

Comment:
The MinGW-based Qt SDK works just fine with these flags, so !win32 looks too heavy a hammer. Disabling these compiler flags needs to be conditional on MSVC, rather than win32 as a whole.